client/components/AIAssistant.js

```diff
@@ -48,13 +48,20 @@ export const AIAssistant = ({ roomUrl }) => {
     "app-message",
     useCallback((ev) => {
       const data = ev?.data;
-      if (data?.kind === "ai-summary") {
-        setSummary(data.data);
+      if (!data) return;
+      const kind = data.kind;
+      const err = data.error;
+      if (err) {
+        playAudioError();
+      }
+      const msg = err ? err : data.data;
+      if (kind === "ai-summary") {
+        setSummary(msg);
```

Comment:
Ah, you might have to rebase now, as I've merged #22. There's no `summary` state anymore.

Reply:
Rebase 
![image](https://media1.giphy.com/media/JtLrtaN4VPoKXJRKGB/giphy.gif?cid=ecf05e47c8b5jybapw5e3x96ksgvvz7wyrc5qkzi9u97bg1f&ep=v1_gifs_search&rid=giphy.gif&ct=g)

Merge-and-squash

![image](https://media1.giphy.com/media/Od0QRnzwRBYmDU3eEO/giphy.gif?cid=ecf05e47gs0woyemh021bkn0t3qeraamu6x35d4c5hlcvyfs&ep=v1_gifs_search&rid=giphy.gif&ct=g)